CMakeLists.txt

```diff
@@ -4,14 +4,36 @@
 
 message("CMakeLists.txt - NimbleSM")
 
-cmake_minimum_required(VERSION 3.12)
+# Needed to specify the version of NimbleSM
+CMAKE_POLICY(SET CMP0048 NEW)
```

Comment:
I think the reason you need this is because CMake's minimum version is required below the project command

Reply:
Done.